.changeset/long-beds-mate.md

```diff
@@ -0,0 +1,26 @@
+---
+'gatsby-plugin-clerk': major
+'@clerk/localizations': major
+'@clerk/nextjs': major
+'@clerk/clerk-react': major
+'@clerk/remix': major
+'@clerk/types': major
+---
+
+Drop deprecations. Migration steps:
+- drop `orgs` jwt claim from session token
+- change type of `withServerAuth()#auth` param to `AuthObject` from `ServerSideAuth` in `gatsby-clerk-plugin`
+    - use `auth.sessionClaims` instead of `auth.claims`
+    - use `AuthObject` properties from `auth`
+- use `publishableKey` instead of `frontendApi`
+- drop `formFieldLabel__emailAddress_phoneNumber` from localization keys
+- drop `formFieldLabel__phoneNumber_username` from localization keys
+- drop `formFieldLabel__emailAddress_phoneNumber_username` from localization keys
+- drop `formFieldInputPlaceholder__emailAddress_phoneNumber` from localization keys
+- drop `formFieldInputPlaceholder__phoneNumber_username` from localization keys
+- drop `formFieldInputPlaceholder__emailAddress_phoneNumber_username` from localization keys
```

Comment:
❓ Does anyone know if those keys are dropped or replaced? 
cc: @panteliselef , @nikosdouvlis

Reply:
I believe the replacements can be found [here](https://github.com/clerk/javascript/blob/73d249192cd89ca2d65cc7e15a9665886e12c2be/packages/clerk-js/src/ui/common/constants.ts#L13)

cc @nikosdouvlis

Reply:
after discussing it with @desiprisg, we should drop them as they were kept to avoid breaking the builds of existing customers.